arm-servicefabric/2016-09-01/swagger/servicefabric.json

```diff
@@ -773,48 +823,19 @@
         }
       }
     },
-    "ClusterPropertiesUpdateParameters": {
-      "description": "Cluster properties update parameters",
+    "ClusterUpdateParameters": {
+      "description": "Cluster update request",
       "properties": {
         "properties": {
           "x-ms-client-flatten": true,
-          "$ref": "#/definitions/ClusterProperties"
+          "$ref": "#/definitions/ClusterPropertiesUpdateParameters"
```

Comment:
Please update the example for PATCH https://github.com/Azure/azure-rest-api-specs/pull/984/files#diff-44c8701ca94084fc58079527bb55d2c3, [ for example `propertyNames` in the PATCH request is no longer required right?]